src/com/twitter/intellij/pants/settings/PantsExecutionSettings.java

```diff
@@ -12,44 +12,38 @@
 import java.util.Objects;
 
 public class PantsExecutionSettings extends ExternalSystemExecutionSettings implements PantsExecutionOptions {
-  private final boolean myWithDependees;
   private final boolean myLibsWithSourcesAndDocs;
   private final boolean myUseIdeaProjectJdk;
   private final boolean myEnableIncrementalImport;
   private final List<String> myTargetSpecs;
 
   private static final List<String> DEFAULT_TARGET_SPECS = Collections.emptyList();
-  private static final boolean DEFAULT_WITH_DEPENDEES = false;
```

Comment:
`DEFAULT_WITH_DEPENDEES` itself is removed because it is never used, but the readability is applied elsewhere.